UefiTestingPkg/FunctionalSystemTests/MpManagement/Driver/MpManagement.c

```diff
@@ -64,6 +64,39 @@ GetProcessorInformation (
   return EFI_SUCCESS;
 }
 
+/**
+  Helper function to query if a processor is enabled.
+
+  @param Mp             MP Services Protocol.
+  @param CpuIndex       Index of processor that is of interest.
+
+  @return TRUE   The processor is enabled.
+  @return FALSE  The processor is disabled.
+**/
+STATIC
+BOOLEAN
+IsProcessorEnabled (
+  IN  EFI_MP_SERVICES_PROTOCOL  *Mp,
+  UINTN                         CpuIndex
+  )
+{
+  EFI_PROCESSOR_INFORMATION CpuInfo;
+  EFI_STATUS                Status;
+
+  if (Mp == NULL) {

```

Comment:
Mp isn't used in this function, is the check on it being NULL just for completeness?

Reply:
Nice catch, it should have been the call to Mp instead of `mMpServices`.